vmd-client/vmmanager.pro

```diff
@@ -0,0 +1,32 @@
+QT += qml quick core
+
+# You can make your code fail to compile if it uses deprecated APIs.
+# In order to do so, uncomment the following line.
+#DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x060000    # disables all the APIs deprecated before Qt 6.0.0
```

Comment:
I'd rather see only specific set of versioned Qt APIs supported in flake.nix defined build environment. Then this can be enabled by default. Or is there any reason to support compilation with deprecated APIs?